src/server/routes/htw.js

```diff
@@ -93,6 +94,65 @@ export function setupHtw(App) {
     )
   })
 
+  App.express.get('/api/wechall/validate', async (req, res) => {
+    const username = req.query.username?.toString() ?? ''
+    const email = req.query.email?.toString() ?? ''
+    const authkey = req.query.authkey?.toString() ?? ''
+
+    if (authkey !== secrets('secret_wechall_authkey')) {
```

Comment:
Another issue: the authkey var is not named secret_ but config_ :)
`config_wechall_authkey`

Reply:
again, thank you for your diligence, this wasn't my best PR 😅 should be fixed in #576